package.json

```diff
@@ -75,6 +75,7 @@
     "jsdom": "22.1.0",
     "lint-staged": "13.2.2",
     "postcss": "8.4.23",
+    "postcss-color-mod-function": "3.0.3",
```

Comment:
![image](https://github.com/rotki/ui-library/assets/2269732/da5d500d-5b64-4ed6-adb2-9f0481173f5e)

can we find an alternative here?

Reply:
removed this, I just figured out we can do it in css directly
https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/color-mix.
What an era to be alive 🤣